.github/workflows/release.yml

```diff
@@ -167,3 +203,4 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
           tags: latest,${{ steps.version.outputs.number }}
+          platform: linux/amd64,linux/arm64
```

Comment:
It looks like the Dockerfile [only adds the authn-linux64 binary](https://github.com/keratin/authn-server/blob/main/Dockerfile#L6). You may need to make both binaries available in the build context and then somehow use [`ARG TARGETPLATFORM`](https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images) to add the correct one?